core/trino-main/src/main/java/io/trino/server/ui/OAuth2WebUiAuthenticationFilter.java

```diff
@@ -119,7 +119,7 @@ public void filter(ContainerRequestContext request)
                 return service.convertTokenToClaims(accessToken.get());
```

Comment:
This is nice. What validates (does the checkstyle for) this?

Reply:
This is Error Prone, though somewhat by accident. It sees that there's no placeholder for the exception in the format string.